modules/misc/news.nix

```
@@ -1381,6 +1381,32 @@ in
           A new module is available: 'services.lieer'.
         '';
       }
+
+      {
+        time = "2020-03-15T16:55:28+00:00";
+        condition = config.programs.firefox.enable;
+        message = ''
+          In preparation of Firefox's dropping support for extension
+          sideloading[1] we now install extensions directly in Firefox
```

```suggestion
          In anticipation of Firefox dropping support for extension
          sideloading[1], we now install extensions directly to Firefox
```

(that `to` change is optional; looks/sounds better to me but no real difference either way)